app/models/event.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+class Event < ApplicationRecord
+  scope :upcoming, ->(n = Date.today.to_s) { where('date > ?', n) }
```

Comment:
- [ ] you should add and = sign so the event that tales place _today_ appears somewhere as well

Reply:
Sorry can not understand what you mean here.